diff --git a/gradle/libs.versions.toml b/gradle/libs.versions.toml
index e25c6a15468..a4c48182e9e 100644
--- a/gradle/libs.versions.toml
+++ b/gradle/libs.versions.toml
@@ -5,7 +5,7 @@ mockito = "5.23.0"
 hamcrest = "3.0"
 jogl = "2.6.0"
 javagiac = "70501"
-gwt = "2.13.1"
+gwt = "2.14.0-SNAPSHOT"
 gwtTreblereel = "202602114"
 elemental2 = "1.3.2"
 murok = "1.0.202604140857"
diff --git a/source/web/settings.gradle.kts b/source/web/settings.gradle.kts
index f3e37938963..06b6f47e386 100644
--- a/source/web/settings.gradle.kts
+++ b/source/web/settings.gradle.kts
@@ -13,6 +13,7 @@ dependencyResolutionManagement {
     repositories {
         maven { url = uri("https://repo.geogebra.net/releases") }
         mavenCentral()
+        mavenLocal()
     }
     versionCatalogs {
         create("libs") {
